docs/content/docs/general/concepts/images/analysis/content_hints.md

```diff
@@ -5,11 +5,11 @@ weight: 5
 ---
 
 
-Anchore Engine includes the ability to read a user-supplied 'hints' file to allow users to override and/or augment the software artifacts that are 
-discovered by Anchore during its image analysis process.  The hints file, if present, contains records that describe a software package characteristics explicitly, 
+Anchore Engine includes the ability to read a user-supplied 'hints' file to allow users to add software artifacts to Anchore's analysis report.  The hints file, if present, contains records that describe a software package characteristics explicitly, 
 and are then added to the software bill of materials (SBOM).  For example, if the owner of a CI/CD container build process knows that there are some 
-software packages installed explicitly in a container image, but Anchore's regular analyzers are either not discovering the software or the analysis of the software 
-package is incomplete, this mechanism can be used to include that information in the image's SBOM, exactly as if the package were discovered normally.
+software packages installed explicitly in a container image, but Anchore's regular analyzers fail to them, this mechanism can be used to include that information in the image's SBOM, exactly as if the packages were discovered normally. 
```

Comment:
"fail to them" seems like a typo. I think you need a verb after "fail to" like "find".

Reply:
also nit: line lengths seem off